datastore.go

```diff
@@ -79,15 +79,25 @@ var DefaultOptions Options
 
 func init() {
 	DefaultOptions = Options{
-		GcDiscardRatio: 0.2,
+		GcDiscardRatio: 0.01,
```

Comment:
0.2 is likely good enough (although 0.1 may be better). We're not going for "delete everything", we're going for "recover space". 0.2 is 20% and leaving <=20% garbage on disk is fine in almost all cases (although it may make sense to expose this).

Dropping this down to 0.01 would force us to do a lot of extra work every time we garbage collect, just to save 1% of space.